### ISSUE TITLE
treegrid and aria-orientation?

### ISSUE THREAD

**Issue by scottaohara** [w3c/aria#1685]
Was reviewing roles that support `aria-orientation` today and I realized that `treegrid` was one such role.  Maybe I am missing something, but this seems like a possible oddity of the role inheriting from both `grid` and `tree`, the latter of which supports `aria-orientation` but not the former.

Is there a use case for `aria-orientation` on `treegrid` that I'm missing?  Or being a bi-directional grid, is this an error?

**Comment by JAWS-test** [comment]
aria-orientation on treegrid only makes sense if we want to allow treegrids where not the rows can be faded in and out, but the columns. But I have never seen such a treegrid. But theoretically it is possible. But then it would also be possible that both rows and columns can be faded in and out, which leads to the fact that we would need aria-orientation=both.

**Comment by JAWS-test** [comment]
Maybe we should also add: 

> Implicit Value for Role: Default for aria-orientation is vertical.
